<launch>
	<arg name="camera"      default="camera"/>
	<arg name="scan_topic" default="scan"/>
	 
	<!--node pkg="b_laserscan" type="b_laserscan_node" name="b_laserscan_node" output="screen">
	 	<param name="scan_height" value="10"/>
	     <param name="output_frame_id" value="/$(arg camera)_depth_frame"/>
	     <param name="range_min" value="0.45"/>
	 	<remap from="image" to="/$(arg camera)/depth/image"/>	 	
		<remap from="scan" to="/$(arg scan_topic)"/>
		<remap from="camera_info" to="/camera/depth/camera_info"/>
	</node-->
	<node pkg="b_orbcamera" type="b_orbcamera_node" name="b_orbcamera_node"  output="screen">
		<rosparam>
            transform_tolerance: 0.01
            min_height: 0.3
            max_height: 1.5

            angle_min: -0.50 #orbbec 
            angle_max: 0.50 #orbbec
            #angle_increment: 0.00159814720973 #for depthimage_to_laserscan is per pix
            angle_increment: 0.0032
            scan_time: 0.033
            range_min: 0.4
            range_max: 8.0
            use_inf: true
            open_skeleton: true

            # Concurrency level, affects number of pointclouds queued for processing and number of threads used
            # 0 : Detect number of cores
            # 1 : Single threaded
            # 2->inf : Parallelism level
            concurrency_level: 1
        </rosparam>
	</node>
	<!--include file="$(find b_orbcamera)/launch/rgb_depth_regist_combine.launch2.xml"></include-->
	
	
	<!--include file="$(find usb_cam)/launch/usb_cam.launch" /-->
<!--	<include file="$(find b_usb_cam)/launch/b_usb_cam.launch" -->
		
<!--	</include-->
	<!--include file="$(find spark_description)/launch/spark_description.launch"/-->
	 
</launch> 
